engine/src/main/java/org/terasology/world/block/tiles/WorldAtlasImpl.java

```diff
@@ -205,7 +224,7 @@ private void buildAtlas() {
 
         int numMipMaps = getNumMipmaps();
         ByteBuffer[] data = createAtlasMipmaps(numMipMaps, TRANSPARENT_COLOR, tiles, "tiles.png");
-        ByteBuffer[] dataNormal = createAtlasMipmaps(numMipMaps, UNIT_Z_COLOR, tilesNormal, "tilesNormal.png");
+        ByteBuffer[] dataNormal = createAtlasMipmaps(numMipMaps, UNIT_Z_COLOR, tilesNormal, "tilesNormal.png", tilesGloss);
```

Comment:
Add the gloss tiles to the alpha channel of the normal map texture.  That way it can get sent in to the shaders without binding another texture and creating the infrastructure to support another bound texture through the layers of shaderland.